src/App.css

```diff
@@ -36,15 +37,13 @@
     display: flex;
     flex-direction: row;
     justify-content: space-evenly;
-    margin-block-start: 20px;
+    margin-block-start: 1rem;
 }
 
 .vs {
     font-size: 2rem;
     font-weight: bold;
-    margin-block-end: 60px;
-    margin-left: 20px;
-    margin-right: 20px;
+    padding: 1.5rem;
```

Comment:
What was the reason for removing "vs" margin, the text does not seem to be centred between the two input boxes.